app/assets/javascripts/fortitude/components/navigationbar.js

```diff
@@ -70,8 +70,10 @@ $(document).on('open.ft.navigationbar', '.navigationbar', function(event) {
   $(document).on('change', '[ft-navigationbar-toggle], [data-ft-navigationbar-toggle]', function(){
     if($(this).prop('checked')){
       $('[ft-shade], [data-ft-shade]').trigger('open.ft.shade');
+      $.screenLock();
```

Comment:
Default is to lock, since that's what the function name implies. If you pass true, it does the same thing, if you pass false or null it unlocks